ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -377,4 +397,37 @@ public void run() {
       }
     }
   }
+
+  private class OptimizerMetricsCalculator implements Runnable {
+    private final long startTime;
+    private final MetricsReporters metricsReporters;
+
+    public OptimizerMetricsCalculator(MetricsReporters metricsReporters) {
+      this.startTime = System.currentTimeMillis();
+      this.metricsReporters = metricsReporters;
+    }
+
+    @Override
+    public void run() {
+      Map<String, OptimizerReport> groupCounters = Maps.newHashMap();
+      // Calculate the total time sum of quotas for all tables under each group.
+      tableManager.listManagedTables().forEach(tableIdentifier -> {
+        TableRuntime tableRuntime = tableManager.getRuntime(tableIdentifier);
+        OptimizerReport optimizerReport = groupCounters.computeIfAbsent(
+            tableRuntime.getOptimizerGroup(),
+            OptimizerReport::new);
+        optimizerReport.optimizerOccupyTime().inc(tableRuntime.getQuotaTime());
+      });
+
+      // Calculate the percentage of time taken by executing tasks for each group
+      // in relation to the total time taken by the optimizer to run.
+      groupCounters.forEach((group, report) -> {
+        int totalThreadCount = listOptimizers(group).stream().mapToInt(Resource::getThreadCount).sum();
+        report.optimizerThreadCount().inc(totalThreadCount);
+        report.optimizerOccupyInterval()
+            .inc(Math.min(System.currentTimeMillis() - startTime, ArcticServiceConstants.QUOTA_LOOK_BACK_TIME));
```

Comment:
The current calculation logic for quotaTime in the process is based on the time spent on optimizing within the last hour. Therefore, the denominator should always be one hour, irrespective of the metric's statistical period.